spinalcordtoolbox/utils/shell.py

```diff
@@ -111,16 +122,25 @@ def _construct_fslview_syntax(viewer, files, colormaps, minmax, opacities, mode)
     return cmd
 
 
-def _construct_fsleyes_syntax(viewer, files, colormaps, minmax, opacities):
+def _construct_fsleyes_syntax(viewer, files, colormaps, im_types, minmax, opacities):
     dict_fsleyes = {'gray': 'greyscale', 'red-yellow': 'red-yellow', 'blue-lightblue': 'blue-lightblue', 'red': 'red',
                     'green': 'green', 'random': 'random', 'hsv': 'hsv', 'subcortical': 'subcortical'}
+    imtypes_colormap = {
+    'anat':    {'fsleyes': 'greyscale' ,  'fslview': 'Greyscale'},
+    'seg':     {'fsleyes': 'red',         'fslview': 'Red'},
+    'softseg': {'fsleyes': 'red-yellow',  'fslview': 'Red-Yellow'},
+    'labels':  {'fsleyes': 'subcortical', 'fslview': 'MGH-Subcortical'},
+    }
```

Comment:
> * Should we store the mapping between image types and colormaps in an other place to avoid repeating the variable in `_construct_fslview_syntax` and `_construct_fsleyes_syntax` ?

Good question! I agree that we could move it somewhere else to avoid duplication.

I think a good place to put it would be underneath the `SUPPORTED_VIEWERS` variable:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/de11527807ffd622763983541f395547f33cfe6f/spinalcordtoolbox/utils/shell.py#L40

This variable is declared outside of any functions, indicating that this is a "global" or "module-level" variable, intended to be used by multiple functions throughout `shell.py`. As well, it has an `ALL_CAPITALS` name, which denotes that it should be treated as a [constant](https://realpython.com/python-constants/).

Given that `imtype_colormaps` serves a similar purpose (module-level, constant), I would say to move the variable there, thenmake it all caps: `IMTYPE_COLORMAPS`.